android_jni/avifandroidjni/src/main/jni/CMakeLists.txt

```diff
@@ -15,9 +15,16 @@ project(avif_android_jni)
 # To override the variables in the libavif subproject:
 # https://stackoverflow.com/questions/3766740/overriding-a-default-option-value-in-cmake-from-a-parent-cmakelists-txt
 set(BUILD_SHARED_LIBS OFF CACHE BOOL "" FORCE)
+
+# To disable libgav1, change the following two variables to OFF.
 set(AVIF_LOCAL_LIBGAV1 ON CACHE BOOL "" FORCE)
 set(AVIF_CODEC_LIBGAV1 ON CACHE BOOL "" FORCE)
 
+# To disable dav1d, change the following two variables to ON.
+set(AVIF_LOCAL_DAV1D OFF CACHE BOOL "" FORCE)
+set(AVIF_CODEC_DAV1D OFF CACHE BOOL "" FORCE)
+
+
```

Comment:
extra line, not sure if it was intentional.

Reply:
Removed.